asset/style/main.css

```diff
@@ -0,0 +1,210 @@
+/* Normalize css code */
+html,
+body,
+div,
+span,
+applet,
+object,
+iframe,
+h1,
+h2,
+h3,
+h4,
+h5,
+h6,
+p,
+blockquote,
+pre,
+a,
+abbr,
+acronym,
+address,
+big,
+cite,
+code,
+del,
+dfn,
+em,
+img,
+ins,
+kbd,
+q,
+s,
+samp,
+small,
+strike,
+strong,
+sub,
+sup,
+tt,
+var,
+b,
+u,
+i,
+center,
+dl,
+dt,
+dd,
+ol,
+ul,
+li,
+fieldset,
+form,
+label,
+legend,
+table,
+caption,
+tbody,
+tfoot,
+thead,
+tr,
+th,
+td,
+article,
+aside,
+canvas,
+details,
+embed,
+figure,
+figcaption,
+footer,
+header,
+hgroup,
+menu,
+nav,
+output,
+ruby,
+section,
+summary,
+time,
+mark,
+audio,
+video {
+  margin: 0;
+  padding: 0;
+  border: 0;
+  font-size: 100%;
+  vertical-align: baseline;
+}
+
+/* HTML5 display-role reset for older browsers */
+article,
+aside,
+details,
+figcaption,
+figure,
+footer,
+header,
+hgroup,
+menu,
+nav,
+section {
+  display: block;
+}
+
+body {
+  line-height: 1;
+}
+
+ol,
+ul {
+  list-style: none;
+}
+
+blockquote,
+q {
+  quotes: none;
+}
+
+blockquote::before,
+blockquote::after,
+q::before,
+q::after {
+  content: '';
+  content: none;
+}
+
+table {
+  border-collapse: collapse;
+  border-spacing: 0;
+}
+
+/* My Style section  */
+header {
+  background-image: url(https://i.postimg.cc/rmSQD83K/headerbg.png);
+  background-repeat: no-repeat;
+  background-size: cover;
+}
+
+/* Navbar styles */
+nav {
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
+}
+
+.navMenu {
+  padding: 10px 12px;
+  font-family:
+
+    'Inter',
+    sans-serif;
+  text-decoration: none;
+  font-weight: 600;
+  font-size: 15px;
+  line-height: 20px;
+  text-align: center;
+  color: #344563;
+}
+
+/* Home section style */
+#homeSection {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  padding: 160px;
+}
```

Comment:
- [ ] Kindly adjust your home section to fit the figma design
## Your design vs Figma

![your_home](https://user-images.githubusercontent.com/69858269/164330485-2ef198f1-81f5-420a-8cf0-c7ab2ae63348.PNG)![figma_home](https://user-images.githubusercontent.com/69858269/164330526-83b093af-d2d7-4463-9d10-30da2c529089.PNG)